Sources/DocCDocumentation/DocCDocumentation.docc/V5_API_Migration_guide.md

```diff
@@ -39,16 +40,15 @@ If you're using the Firebase SDK, you'll need to follow [these instructions](htt
 Version 5.0 of the SDK  deprecates the term "Observer Mode" (and the APIs where this term was used), and replaces it
 with `PurchasesAreCompletedBy` (either RevenueCat or your app).
 
-Version 5.0 of the SDK also introduces support for recording purchases made directly by your app calling StoreKit 2. 
+Version 5.0 of the SDK also introduces support for tracking purchases made directly by your app calling StoreKit 2.
 
-You can enable it when configuring the SDK:
+If you're using RevenueCat only to track purchases, and you have your own implementation of StoreKit to make purchases, you will need to explicitly configure the SDK.
```

Comment:
Nit: all developers need to explicitly configure the SDK, these developers need to explicitly define their SK version when calling configure()

```suggestion
If you're using RevenueCat only to track purchases, and you have your own implementation of StoreKit to make purchases, you will need to explicitly provide your StoreKit version when you configure the SDK.
```